### PR TITLE
Quickstart auto-open after init

### PR DESCRIPTION
**Pull request by ptrdvrk**
This update is meant for the UX testing exercise. We should have similar functionality for GA, but only open the Quickstart the first time.

### CODE REVIEW

File: src/tree/milestoneTreeDataProvider.ts [hunk 1]
```diff
@@ -56,6 +56,10 @@ export class MilestoneTreeDataProvider implements vscode.TreeDataProvider<vscode
     return element;
   }
 
+  public getParent(): Thenable<vscode.TreeItem | null> {
```
Comment (apotterri) [inline src/tree/milestoneTreeDataProvider.ts:59]: Is this necessary? It doesn't seem like it's doing much....

Comment (ptrdvrk) [reply src/tree/milestoneTreeDataProvider.ts:59]: Without this the milestoneTree.reveal() in extension.ts does not work, errors on missing function getParent().

Comment (apotterri) [reply src/tree/milestoneTreeDataProvider.ts:59]: Do you understand why that is? Why wasn't it necessary before your change?

**Review by apotterri — CHANGES_REQUESTED** [review]
The two commits in this PR should get squashed together, which could be done when the PR gets merged. However, they have different semantic release prefixes. Which is correct?

**Review by ptrdvrk — COMMENTED** [review]
> The two commits in this PR should get squashed together, which could be done when the PR gets merged. However, they have different semantic release prefixes. Which is correct?

The second commit with the "misc:" prefix just updated the comments. It can be removed.

**Review by apotterri — APPROVED** [review]

**Pull request merged**